<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="mainWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="menulabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Меню&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" rowspan="9">
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="page_3">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="letter_abc">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>550</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>513</height>
               </size>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>318</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="nextabcButton">
                <property name="maximumSize">
                 <size>
                  <width>91</width>
                  <height>23</height>
                 </size>
                </property>
                <property name="text">
                 <string>Далі</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QPushButton" name="exitabcButton">
                <property name="maximumSize">
                 <size>
                  <width>71</width>
                  <height>23</height>
                 </size>
                </property>
                <property name="text">
                 <string>Завершити</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_4"/>
          <widget class="QWidget" name="page_5">
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>210</y>
              <width>291</width>
              <height>181</height>
             </rect>
            </property>
            <property name="text">
             <string>Дана програма створенна на замовленням Калуської с</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="abclabel">
            <property name="text">
             <string>Розділ 1.Абетка</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="abcButton">
              <property name="text">
               <string>Абетка</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="studyButton">
              <property name="text">
               <string>Вивчення</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="testabcButton">
              <property name="text">
               <string>Тест</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="themelabel">
          <property name="text">
           <string>Розділ 2. Теми</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Овочі</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="test1vegetableButton">
              <property name="text">
               <string>Тест1. Впиши слово</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="test2vegetableButton">
              <property name="text">
               <string>Тест2. Вибери слово</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Фрукти</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="test1fruitButton">
            <property name="text">
             <string>Тест 1. Впиши слово</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="test2fruitButton">
            <property name="text">
             <string>Тест2. Вибери слово</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="QPushButton" name="test1animalsButton">
            <property name="text">
             <string>Тест1. Впиши слово</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="test2animalsButton">
            <property name="text">
             <string>Тест2. Вибери слово</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Тварини</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>141</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="usr_Bttn">
          <property name="text">
           <string>Кабінет користувача</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QPushButton" name="outputButton">
          <property name="text">
           <string>Вихід</string>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>menulabel</zorder>
       <zorder>stackedWidget</zorder>
       <zorder>outputButton</zorder>
       <zorder>label_2</zorder>
       <zorder>test1fruitButton</zorder>
       <zorder>test2fruitButton</zorder>
       <zorder>themelabel</zorder>
       <zorder>label_3</zorder>
       <zorder>test1animalsButton</zorder>
       <zorder>test2animalsButton</zorder>
       <zorder>label_3</zorder>
       <zorder>usr_Bttn</zorder>
       <zorder>verticalSpacer</zorder>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QPushButton" name="inputButton">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>430</y>
          <width>161</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Ввійти</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>60</y>
          <width>421</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Times New Roman'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Навчальна програма для дітей дошкільного віку&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;з вадами слуху&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>170</y>
          <width>391</width>
          <height>171</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="surnamelabel">
             <property name="text">
              <string>Прізвище</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="surname"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="userNamelabel">
               <property name="minimumSize">
                <size>
                 <width>51</width>
                 <height>16</height>
                </size>
               </property>
               <property name="text">
                <string>Ім'я</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="userName"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="sexlabel">
             <property name="text">
              <string>стать</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Buttonman">
             <property name="text">
              <string>чоловік</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Buttonwoman">
             <property name="text">
              <string>жінка</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="agelabel">
             <property name="text">
              <string>Вік</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="age"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="defectlabel">
             <property name="text">
              <string>Вада слуху</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Buttonyes">
             <property name="text">
              <string>є</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup_2</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="buttonno">
             <property name="text">
              <string>немає</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">buttonGroup_2</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
   <zorder>outputButton</zorder>
   <zorder>mainWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
